shell/app/common/components/tiled-filter.tsx

```diff
@@ -187,4 +185,77 @@ const TiledFilter = (props: IProps) => {
   );
 };
 
+interface IFieldItemProps {
+  field: IField;
+  labelWidth: number | string;
+  value: string[] | string;
+  onChangeItem: (val: string, field: IField, forceUpdate?: boolean) => void;
+}
+
+const SelectFieldItem = (props: IFieldItemProps) => {
+  const { field, labelWidth, value, onChangeItem } = props;
+  const { type, label, options, showSearch, multiple } = field;
+
+  let Comp: React.ReactNode = null;
+  const _value: string[] = multiple ? value : [value];
+  switch (type) {
+    case 'select':
+      Comp = (
```

Comment:
it seems `Comp` no need Capitalize

Reply:
Comp return as a Component or null, Capitalize as a regular Component